node/bft-consensus/src/lib.rs

```diff
@@ -30,17 +30,19 @@ use narwhal_crypto::NetworkKeyPair;
 use narwhal_executor::ExecutionState;
 use narwhal_node::{primary_node::PrimaryNode, worker_node::WorkerNode, NodeStorage};
 use narwhal_types::{Batch, ConsensusOutput};
-use std::sync::Arc;
+use std::{path::PathBuf, sync::Arc};
 use thiserror::Error;
 use tracing::*;
 
+use aleo_std::aleo_dir;
 use snarkos_node_consensus::Consensus as AleoConsensus;
 use snarkos_node_messages::{Data, Message, NewBlock};
 use snarkos_node_router::Router;
 use snarkos_node_tcp::protocols::Writing;
 use snarkvm::prelude::{ConsensusStorage, Network};
 
 pub struct BftConsensus<N: Network, C: ConsensusStorage<N>> {
+    // TODO(nkls): remove this
```

Comment:
are we sure we'll want to remove the numeric primary ID?

Reply:
I _think_ it's only used for dev logs currently, so don't see why not?

Reply:
Not after this PR 😄 (it's in `dev`).

Reply:
as long as we keep `dev` in the end :detective: